### ISSUE TITLE
Error downloading packages' when building fedora-20

### ISSUE THREAD

**Issue by mcandre** [dozzie/yumbootstrap#15]
Could we add more mirrors to the fedora-20 and older scripts? I think the packages have moved to archival since fedora 22 was released.

```
mkdir /chroot && \
yumbootstrap --verbose fedora-20 /chroot
...
Error downloading packages:
  kpartx-0.4.9-56.fc20.1.x86_64: [Errno 256] No more mirrors to try.
  libxml2-2.9.1-4.fc20.x86_64: [Errno 256] No more mirrors to try.
  nss-sysinit-3.18.0-1.fc20.x86_64: [Errno 256] No more mirrors to try.

"/usr/bin/yum" exited with code 1
```

**Comment by dozzie** [comment]
The packages for Fedora 20 were not moved to archive yet. It seems that the
mirror you were redirected to had some packages missing.

On the other hand, mirrors I have checked all have "updates" repository broken
(damaged bzip2 file; *compressed file ended before the logical end-of-stream
was detected*). Most probably the upstream is broken, which is out of my
reach.

Fedora 18 and 19 indeed were moved to archive. URLs updated, I'll push commits
soon.
